hwconf/hw_Cheap_FOCer_2.h

```diff
@@ -20,7 +20,12 @@
 #ifndef HW_Cheap_FOCer_2
 #define HW_Cheap_FOCer_2
 
+
+#ifdef CFOC_IS_V09
+#define HW_NAME					"Cheap_FOCer_2_v0.9"
```

Comment:
The dot will not work with vesc tool, it has to be the same name as the directory name for vesc tool to list the correct firmware if the firmware is to be included in vesc tool tool.

Reply:
Gotcha, will fix it